sdks/java/core/src/main/java/org/apache/beam/sdk/AggregatorPipelineExtractor.java

```diff
@@ -25,7 +25,6 @@
 import org.apache.beam.sdk.Pipeline.PipelineVisitor;
 import org.apache.beam.sdk.runners.TransformHierarchy;
 import org.apache.beam.sdk.transforms.Aggregator;
-import org.apache.beam.sdk.transforms.AggregatorRetriever;
 import org.apache.beam.sdk.transforms.PTransform;
 import org.apache.beam.sdk.transforms.ParDo;
 import org.apache.beam.sdk.values.PValue;
```

Comment:
Is this file still needed? If so, please `@Deprecated` the main class and comment why.